### PR TITLE
modrinth-app: init at 0.5.4

### PR DESCRIPTION
**Pull request by NotAShelf**
funny mod manager

### CODE REVIEW

**Comment by NotAShelf** [comment]
didn't edit the readme because formatting

**Comment by NotAShelf** [comment]
CI is failing due to insecure openssl 1.1 dependency - nothing I can do about that unfortunately

File: pkgs/modrinth-app/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# https://modrinth.com/app
+{
+  lib,
+  stdenv,
```
Comment (fufexan) [inline pkgs/modrinth-app/default.nix:4]: Let's use `stdenvNoCC`.

**Comment by fufexan** [comment]
Also if CI fails, add `NIXPKGS_ALLOW_INSECURE=1` in the CI file.

**Review by fufexan — APPROVED** [review]

**Pull request merged**